content/docs/use-cases/model-registry.md

```diff
@@ -0,0 +1,64 @@
+# Machine Learning Model Registry
+
+A **model registry** is a repository to catalog ML models and their versions.
```

Comment:
`is a repository` - we start with being DVC-specific?

repository -> tool, service, place ...

Reply:
"repository" has a general meaning too 🙂 which is what we were going for here.

<img width="752" alt="image" src="https://user-images.githubusercontent.com/1477535/165796063-b7ed5077-6cb2-4e8a-94b9-b45097da7806.png">

Reply:
in this case it's clearly overloaded

Reply:
> repository -> tool, service, place

OK I did change this to "construct" after all since "repo"'s definition (computing context) may imply "central" per ☝🏼 (and we want to avoid emphasizing that aspect per https://github.com/iterative/dvc.org/pull/3333#discussion_r861087231).